easybuild/framework/easystack.py

```diff
@@ -148,8 +148,6 @@ def parse(filepath):
                         raise EasyBuildError("Incorrect toolchain specification for '%s' in %s, too many parts: %s",
                                              name, filepath, toolchain_parts)
 
-                check_version(toolchain_version, "software %s (with %s toolchain)" % (name, toolchain_name))
```

Comment:
@branfosj Why remove this check for the toolchain version?

Reply:
`toolchain_version` has either been set to `''` or one item from the return of a `split()` being called on the string `toolchain`.

Reply:
Right, good point.

So this is good to go in, but I'll squeeze in another test case to check that a toolchain version like `2021.03` is handled correctly...